guides/plug.md

```diff
@@ -53,26 +57,26 @@ defmodule HelloWeb.Endpoint do
 end
 ```
 
-Function plugs are plugged by passing the function name as an atom. To try the Plug out, go back to your browser and fetch [http://localhost:4000](http://localhost:4000). You should see something like this printed in your terminal:
+Function plugs are plugged by passing the function name as an atom. To try the plug out, go back to your browser and fetch [http://localhost:4000](http://localhost:4000). You should see something like this printed in your shell terminal:
 
 ```console
 Verb: "GET"
 Host: "localhost"
 Headers: [...]
 ```
 
-Our Plug simply prints information from the connection. Although our initial Plug is very simple, you can virtually do anything you want inside of it. To learn about all fields available in the connection and all of the functionality associated to it, [see the documentation for Plug.Conn](https://hexdocs.pm/plug/Plug.Conn.html).
+Our plug simply prints information from the connection. Although our initial Plug is very simple, you can virtually do anything you want inside of it. To learn about all fields available in the connection and all of the functionality associated to it, see the [documentation for `Plug.Conn`](https://hexdocs.pm/plug/Plug.Conn.html).
 
-Now let's look at the other flavor plugs come in, module plugs.
+Now let's look at the other plug variant, the module plugs.
 
-## Module Plugs
+## Module plugs
 
-Module plugs are another type of Plug that let us define a connection transformation in a module. The module only needs to implement two functions:
+Module plugs are another type of plug that let us define a connection transformation in a module. The module only needs to implement two functions:
 
 - `init/1` which initializes any arguments or options to be passed to `call/2`
 - `call/2` which carries out the connection transformation. `call/2` is just a function plug that we saw earlier
 
-To see this in action, let's write a module plug that puts the `:locale` key and value into the connection assign for downstream use in other plugs, controller actions, and our views. Put the contents above to a file named "lib/hello_web/plugs/locale.ex":
+To see this in action, let's write a module plug that puts the `:locale` key and value into the connection assign for downstream use in other plugs, controller actions, and our views. Put the contents bellow to a file named `lib/hello_web/plugs/locale.ex`:
```

Comment:
> Put the contents bellow to a file named `lib/hello_web/plugs/locale.ex`:

s/bellow/below/

Maybe "in a file" instead of "to a file" too.

Reply:
Sure. Thank you